Letterbook.Core/PostService.cs

```diff
@@ -23,16 +23,18 @@ public class PostService : IAuthzPostService, IPostService
 	private readonly IPostAdapter _posts;
 	private readonly IPostEventPublisher _postEventPublisher;
 	private readonly IActivityPubClient _apClient;
+	private readonly IEnumerable<IContentSanitizer> _sanitizers;
 	private Uuid7 _profileId;
 	private IEnumerable<Claim> _claims = null!;
 
 	public PostService(ILogger<PostService> logger, IOptions<CoreOptions> options,
-		IPostAdapter posts, IPostEventPublisher postEventPublisher, IActivityPubClient apClient)
+		IPostAdapter posts, IPostEventPublisher postEventPublisher, IActivityPubClient apClient, IEnumerable<IContentSanitizer> sanitizers)
```

Comment:
Note the `IEnumerable<IContentSanitizer>`

If multiple factories for the same type are registered, this gives you all of them. So adding support for new content types just requires defining and registering new sanitizers